### ISSUE TITLE
missing tile when images are just loaded

### ISSUE THREAD

**Issue by kswang1029** [CARTAvis/carta-frontend#2026]
**Describe the bug**
This can be reproduced (more reliably) with the python scripting interface. This example script is like 

```
from carta.session import Session

url_with_token = 'http://localhost:3002/?token=0E6F1766-A24C-4821-829C-A19003D0BEC0'
session_id = 911675481

image_file_0 = '/Users/kswang/carta_image_pool/set_imagePyramid_2d/cluster_00128.fits'
image_file_1 = '/Users/kswang/carta_image_pool/set_imagePyramid_2d/cluster_00512.fits'

session = Session.interact(url_with_token, session_id)
img0 = session.open_image(image_file_0)
img1 = session.append_image(image_file_1)

img1.set_spatial_matching(True)
session.save_rendered_view('test.png')
```

Ideally we should see the two images in the multi-panel view and all images are properly rendered. However, with the scripting interface, we will see a tile from the 2nd image is missing. The missing one may be at a different tile location.

![test](https://user-images.githubusercontent.com/20819712/193723906-11400d1a-d806-47c9-aab0-0cdae7145679.png)

If we just select the two images via the GUI and load them at once, there is no missing tile (as far as I can repeat the success apparently).

This might have the same root cause as https://github.com/CARTAvis/carta-frontend/issues/954

**To Reproduce**
1. try the above python script to reproduce the issue (hopefully)
2. try to load the same two images via GUI to see the two images rendered successfully

**Expected behavior**
This issue is more critical for the python scripting interface if we use it to generate publication plots. By all means we should ensure images are rendered properly on load. 

**Platform info (please complete the following information):**
 - OS [e.g. macOS Monterey]: macOS Monterey
 - Browser [e.g. chrome, safari, electron app]: electron app
 - Browser version [e.g. 22]: v3-stable electron app
 - Backend branch [e.g. dev, v3b2 release]: v3
 - Frontend branch [e.g. dev, v3b2 release]: v3

**Comment by kswang1029** [comment]
@YuHsuan-Hwang @crocka @pford do you think #2219 also fixes this issue? (via tile_sync_id?)

**Comment by YuHsuan-Hwang** [comment]
Fixed via #2219.
